api/src/main/java/bio/terra/tanagra/service/underlay/UnderlayConversion.java

```diff
@@ -541,22 +561,30 @@ private static ColumnId convert(bio.terra.tanagra.proto.underlay.ColumnId column
         .build();
   }
 
+  private static TableFilter convert(
+      bio.terra.tanagra.proto.underlay.TableFilter tableFilterProto,
+      Map<ColumnId, Column> columns) {
+    switch (tableFilterProto.getFilterCase()) {
+      case BINARY_COLUMN_FILTER:
+        return TableFilter.builder()
+            .binaryColumnFilter(convert(tableFilterProto.getBinaryColumnFilter(), columns))
+            .build();
+      case FILTER_NOT_SET:
```

Comment:
nit: This is treated equivalently to `null` elsewhere: https://github.com/DataBiosphere/tanagra/blob/a9ad6ff4045b30a6e204c519b733c69961e31228/api/src/main/java/bio/terra/tanagra/service/underlay/UnderlayConversion.java#L189

Reply:
Thanks for the catch, changed the validation switch statement to also throw on `FILTER_NOT_SET` so they're consistent.